sqlmesh/core/loader.py

```diff
@@ -594,6 +603,21 @@ def _load_metrics(self) -> UniqueKeyDict[str, MetricMeta]:
 
         return metrics
 
+    def _load_linting_rules(self) -> RuleSet:
+        rs = None
+        for path in self._glob_paths(
+            self.config_path / c.LINTER,
+            ignore_patterns=self.config.ignore_patterns,
+            extension=".py",
+        ):
+            if os.path.getsize(path):
+                module = import_python_file(path, self.config_path)
+                if hasattr(module, "USER_RULES"):
+                    rs = module.USER_RULES
+                    break
+
+        return rs or RuleSet()
```

Comment:
[Nit] I'd consider returning early in L616 to avoid the `rs = None` + `rs or RuleSet()` pattern.